### ISSUE TITLE
AttributeError: 'Stockfish' object has no attribute 'stockfish'

### ISSUE THREAD

**Issue by mikhailhaimovich** [zhelyabuzhsky/stockfish#34]
\stockfish\models.py", line 271, in __del__
    self.stockfish.kill()

Reference failed

**Comment by mikhailhaimovich** [comment]
I am sure that I am missing something; I guess I just need instructions in simpler terms. Apologies!

**Comment by zhelyabuzhsky** [comment]
Hello )

How did you get this error?

**Comment by mikhailhaimovich** [comment]
Hello!

I got this error after installing Pre-req files GabrieleMaurina chess-cheat and attempting to launch

python -m chess-cheat

**Comment by zhelyabuzhsky** [comment]
Let's ask @GabrieleMaurina =)

**Comment by mikhailhaimovich** [comment]
Makes sense)

I started by asking you because I had seen others with a similar question

**Comment by zhelyabuzhsky** [comment]
> I started by asking you because I had seen others with a similar question

give me the url of issue please

**Comment by mikhailhaimovich** [comment]
https://github.com/zhelyabuzhsky/stockfish/issues/32

**Comment by GabrieleMaurina** [comment]
I am not sure what is going on here. I see in [#32](https://github.com/zhelyabuzhsky/stockfish/issues/32#issue-638462424) the problem is related to the engine path. However in my code, I assume the engine is installed globally. Indeed Stockfish is initialized with something like 
```python
stockfish = Stockfish()
```
without specifying any path. So perhaps that could be your problem too?

**Comment by zhelyabuzhsky** [comment]
@mikhailhaimovich is it still relevant?
